src/test/java/org/itsallcode/io/TestCapturingOuputStream.java

```diff
@@ -55,4 +70,39 @@ void testWithBufferedOutputStreamOnTopOfSystemOut() throws IOException
             System.setOut(previousOut);
         }
     }
+
+    @Test
+    void testOutputForwardedWhenCapturing() throws IOException
+    {
+        final OutputStream delegate = mock(OutputStream.class);
+        final CapturingOutputStream stream = new CapturingOutputStream(delegate);
+        stream.capture();
+        final byte[] content = "content".getBytes();
+        stream.write(content);
+        stream.close();
+        verify(delegate).write(content, 0, 7);
+    }
+
+    @Test
+    void testOutputForwardedWhenNotCapturing() throws IOException
+    {
+        final OutputStream delegate = mock(OutputStream.class);
+        final CapturingOutputStream stream = new CapturingOutputStream(delegate);
+        final byte[] content = "content".getBytes();
+        stream.write(content);
+        stream.close();
+        verify(delegate).write(content, 0, 7);
+    }
+
+    @Test
+    void testOutputNotForwardedWhenCapturingInMuteMode() throws IOException
+    {
+        final OutputStream delegate = mock(OutputStream.class);
+        final CapturingOutputStream stream = new CapturingOutputStream(delegate);
+        stream.captureMuted();
+        final byte[] content = "content\n".getBytes();
```

Comment:
Is there a special reason why you add the newline here in contrast to the tests above?

Reply:
No. I fixed this. ✅